Backpack/Icon/Classes/BPKIconView.m

```diff
@@ -0,0 +1,71 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#import "BPKIconView.h"
+
+#import "BPKColor.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BPKIconView()
+@property(nonatomic, assign) BPKIconSize size;
+
+- (void)setUp;
+@end
+
+@implementation BPKIconView
+
+- (instancetype)initWithIconName:(BPKIconName)iconName size:(BPKIconSize)size {
+    CGSize displaySize = [BPKIcon concreteSizeForIconSize:size];
+
+    self = [super initWithFrame:CGRectMake(0, 0, displaySize.width, displaySize.height)];
+
+    if (self) {
+        self.size = size;
```

Comment:
Can we be explicit about the content mode of the image view? Scale to fit UIViewContentModeScaleAspectFit so it won’t get cropped?

Reply:
Couldn't hurt